test/test_profiler.py

```diff
@@ -33,13 +33,15 @@ def test_mem_leak(self):
             torch.cuda.empty_cache()
             last_rss.append(p.memory_info().rss)
 
-        max_diff = -1
-        for idx in range(1, len(last_rss)):
-            max_diff = max(max_diff, last_rss[idx] - last_rss[idx - 1])
-
-        # with CUDA events leaking the increase in memory was ~7 MB,
-        # using much smaller threshold but not zero to reduce flakiness
-        self.assertTrue(max_diff < 100 * 1024)
+        # with CUDA events leaking the increase in memory was ~7 MB between
+        # profiler invocations above
+        is_increasing = True
+        for idx in range(len(last_rss)):
+            if idx == 0:
+                continue
+            is_increasing = is_increasing and (last_rss[idx] > last_rss[idx - 1])
+        self.assertTrue(not is_increasing,
```

Comment:
```suggestion
        self.assertTrue(is_increasing,
```

Reply:
the condition should be that it is not increasing

Reply:
Hmm, then there are definitely a leak:
```
 memory usage is increasing, deque([1951924224, 1951936512, 1951952896, 1951977472, 1951985664], maxlen=5)
```